SQLExecutorComponent/src/main/java/com/astronomvm/sqlExecutor/SQLExecutorComponent.java

```diff
@@ -0,0 +1,111 @@
+package com.astronomvm.sqlExecutor;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this package name to match the regular expression '^[a-z_]+(\.[a-z_][a-z0-9_]*)*$'. ([squid:S00120](https://sonarcloud.io/organizations/ahmedriahi-github/rules#rule_key=squid%3AS00120))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=AhmedRiahi_AstronomVM&pullRequest=22&issues=AWZfQr15DZywvPe4LfuN&open=AWZfQr15DZywvPe4LfuN)

[ISSUE_KEY]: AWZfQr15DZywvPe4LfuN